### PR TITLE
EZP-30814: Removed Twig deprecations

### PR DESCRIPTION
**Pull request by adamwojs**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-30814](https://jira.ez.no/browse/EZP-30814)
| **Type**           | Improvement
| **Target version** | `master`
| **BC breaks**      | no
| **Doc needed**     | no

Continuation of https://github.com/ezsystems/ezpublish-kernel/pull/2708:

* Replaced `spaceless` tag usages in favour of `spaceless` filter
* Removed `for ... if ` statements

**TODO**:
- [X] Implement feature / fix a bug.
- [ ] ~Implement tests + specs and passing (`$ composer test`)~
- [X] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [ ] Ask for Code Review.

### CODE REVIEW

**Comment by andrerom** [comment]
applicable to stable branches as well to get rid of deprecation notices?

**Comment by adamwojs** [comment]
@andrerom "Yes" in general, but not applicable in this case: `src/Resources/views/fields/content_fields.html.twig` doesn't exist on 0.9.X.

File: src/Resources/views/fields/content_fields.html.twig [hunk 1]
```diff
@@ -3,22 +3,24 @@
 {% extends "EzPublishCoreBundle::content_fields.html.twig" %}
 
 {% block ezobjectrelationlist_field %}
-    {% spaceless %}
+    {% apply spaceless %} %}
```
Comment (emodric) [inline src/Resources/views/fields/content_fields.html.twig]: There's an extra `%}` here and in other places below.

Comment (adamwojs) [reply src/Resources/views/fields/content_fields.html.twig]: Good catch, thank you! Follow up PR: https://github.com/ezsystems/ezplatform-http-cache/pull/97

**Pull request merged**